### DISCUSSION TITLE
Change Base URL?

### DISCUSSION THREAD

**Discussion by ariefsn** [AzuraCast/AzuraCast#4336] [Q&A] [answered]
Hello,
How to change base URL?
So my case like this, let say I have IP 111:11:11:11 
I have install azuracast with docker, its run on port 80,
111:11:11:11 ==> my azuracast app

then I want add new app in root (port 80) and my azuracast on different path.
so become like this:
111:11:11:11 ==> my new app
111:11:11:11/azuracast ==> my azuracast app

and the problem is, all baseUrl still pointing on 111:11:11:11,
how to change to 111:11:11:11/azuracast

thank you...

**Comment by BusterNeece** [comment, marked as answer]
@ariefsn It is not currently possible or recommended to run AzuraCast in a subfolder different from the main root of a domain or subdomain.

However, if you use a domain name and can manage its DNS records, you can install AzuraCast on its own subdomain without any difficulty at all; this is by far the most common setup for AzuraCast on radio stations.

**Reply by ariefsn** [reply]
ok thank you,
seems like have to buy domain :)

**Reply by umeshpunia** [reply]
where all files stored?
